.github/actions/on_device_tests/action.yaml

```diff
@@ -1,9 +1,73 @@
-name: On Host Tests
-description: Runs on-host tests.
+name: On Device Test
+description: Runs on-device tests.
+
 runs:
   using: "composite"
   steps:
-    - name: Run On-Device Tests
+    - name: Install Requirements
+      run: |
+        pip3 install grpcio==1.38.0 grpcio-tools==1.38.0
+      shell: bash
+    - name: Generate gRPC files
+      run: |
+        python -m grpc_tools.protoc -Itools/ --python_out=cobalt/tools/ --grpc_python_out=cobalt/tools/ cobalt/tools/on_device_tests_gateway.proto
```

Comment:
I am not sure if this relative path is correct, the absolute path is `${GITHUB_WORKSPACE}/src/cobalt/tools`

Reply:
+1 to using absolute path here